db/seeds.rb

```diff
@@ -6,23 +6,30 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-categories = Category.create([{title: "Apparel"},{title: "Snowboards"}])
+categories = Category.create( [ { title: "Apparel" },{ title: "Snowboards" } ] )
```

Comment:
Useless assignment to variable - `categories`.<br>Space missing after comma.<br>Space inside square brackets detected.<br>Space inside parentheses detected.